tracker_test.go

```diff
@@ -0,0 +1,64 @@
+// +build integration
+
+package main
+
+import (
+	"testing"
+
+	"github.com/almighty/almighty-core/app"
+	"github.com/almighty/almighty-core/app/test"
+	"github.com/almighty/almighty-core/models"
+)
+
+func TestCreateTracker(t *testing.T) {
+	ts := models.NewGormTransactionSupport(db)
+	repo := models.NewTrackerRepository(ts)
+	controller := TrackerController{ts: ts, tRepository: repo}
+	payload := app.CreateTrackerPayload{
+		URL:  "http://issues.jboss.com",
+		Type: "jira",
+	}
+
+	_, created := test.CreateTrackerCreated(t, nil, nil, &controller, &payload)
+	if created.ID == "" {
+		t.Error("no id")
+	}
+}
+
```

Comment:
How about tests for the other status codes for the Resources; NOT_FOUND, BAD_REQUEST etc..